.hooks/validate-mdx.js

```diff
@@ -1,70 +1,64 @@
-const { readFile } = require('fs-extra');
-const mdx = require('@mdx-js/mdx');
+const { spawn } = require('child_process');
+const path = require('path');
 
+const files = process.argv.slice(2);
 
-const NO_HTML = "<!-- NOT_HTML_DOC -->";
-const YES_HTML = "<!-- HTML_DOC -->";
-
-function isHtmlDoc(content) {
-    return content.startsWith(YES_HTML) ||
-        (!content.startsWith(NO_HTML) && (
-            content.startsWith("<p>") ||
-            content.startsWith("<!DOCTYPE html>") ||
-            (content.includes("<thead>") && content.includes("<tbody>"))
-        ));
-}
-
-
-function fixMdx(readmeContent) {
-    // copied from: https://github.com/demisto/content-docs/blob/2402bd1ab1a71f5bf1a23e1028df6ce3b2729cbb/content-repo/mdx_utils.py#L11
-    // to use the same logic as we have in the content-docs build
-    let txt = readmeContent;
-
-    const replaceTuples = [
-        [/<br>(?!<\/br>)/gi, "<br/>"],
-        [/<hr>(?!<\/hr>)/gi, "<hr/>"],
-        [/<pre>/gi, "<pre>{`"],
-        [/<\/pre>/gi, "`}</pre>"]
-    ];
-
-    replaceTuples.forEach(([oldPattern, newValue]) => {
-        txt = txt.replace(oldPattern, newValue);
+async function runValidator(file, heapLimit = 2048) {
+    return new Promise((resolve, reject) => {
+        let stderrData = '';
+
+        const child = spawn(
+            'node',
+            [`--max-old-space-size=${heapLimit}`, path.join(__dirname, 'parse-mdx-single.js'), file],
+            { stdio: ['ignore', 'inherit', 'pipe'] }
+        );
+
+        child.stderr.on('data', (data) => {
+            stderrData += data.toString();
+        });
+
+        child.on('exit', (code, signal) => {
+            const oomExitCodes = [134, 137];
+            const oomSignals = ['SIGKILL', 'SIGABRT'];
+            const isOOM =
+                oomExitCodes.includes(code) ||
+                oomSignals.includes(signal) ||
+                stderrData.includes('JavaScript heap out of memory') ||
+                stderrData.includes('Allocation failed - JavaScript heap out of memory');
```

Comment:
```suggestion
                stderrData.includes('JavaScript heap out of memory');
```
The sentences are contained within one another.